eng/tools/TypeSpecValidation/src/rules/format.ts

```diff
@@ -8,9 +8,11 @@ export class FormatRule implements Rule {
 
   async execute(folder: string): Promise<RuleResult> {
     // Format parent folder to include shared files
-    let [stdOutput, errorOutput] = await runCmd(`npx tsp format ../**/*.tsp`, folder);
 
-    let success = errorOutput == null ? false : true;
+    let [err, stdOutput, errorOutput] = await runCmd(`npx tsp format ../**/*.tsp`, folder);
+    // Failing on both err and errorOutput because of known bug in tsp format where it returns 0 on failed formatting
+    // https://github.com/microsoft/typespec/issues/2323
+    let success = err || errorOutput ? false : true;
```

Comment:
```suggestion
    let success = !err && !errorOutput;
```

This is more succinct and I think equivalent?